projects/components/src/input/input.component.scss

```diff
@@ -9,28 +9,43 @@
     }
 
     .border {
+      .mat-form-field-wrapper {
```

Comment:
you can put these names in the same line. also using the scss incorrectly. You are not using 'use' instead of 'include'